lib/generators/administrate/routes/routes_generator.rb

```diff
@@ -49,15 +49,15 @@ def dashboard_resources
       end
 
       def valid_dashboard_models
-        database_models - invalid_database_models
+        database_models - invalid_dashboard_models
       end
 
       def database_models
         ActiveRecord::Base.descendants.reject(&:abstract_class?)
       end
 
-      def invalid_database_models
-        models_without_tables + namespaced_models + unnamed_constants
+      def invalid_dashboard_models
```

Comment:
This is a nice change!

Reply:
Yeah, the inconsistent naming was bothering me too.